tests/test_masks.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from src.masks import get_mask_account
+from src.masks import get_mask_card_number
+
+
+def test_get_mask_account(number_of_account_for_masks: str) -> None:
+    # Проверка на правильность маскировки счета
+    assert get_mask_account("12345678901234567890") == number_of_account_for_masks
```

Comment:
По тесту не понятно как работает функция - для этого нужно идти в фикстуру и смотреть как она устроена. 

Вообще в фикстуры переноси только те данные, которые можно переиспользовать, либо те, которые занимают много места в самом тесте

В данном случае правильнее будет так:
```suggestion
    assert get_mask_account("12345678901234567890") == '**7890'
```

Reply:
исправил, делал так только потому что в задании было написано делать фикстуры для каждого теста, хотя действительно лишняя работа.